tests/select/test_drop_over_night_pv.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime
```

Comment:
Great job on these! I also recommend having the tests with the PVOutput datapipe `pvoutput_datapipe` to check it works for both that we use